zebra/zapi_msg.c

```diff
@@ -2380,6 +2381,7 @@ static inline void zread_iptable(ZAPI_HANDLER_ARGS)
 	zpi.interface_name_list = list_new();
 	zpi.sock = client->sock;
 	zpi.vrf_id = zvrf->vrf->vrf_id;
+	zpi.ns_id = zvrf->zns->ns_id;
 	STREAM_GETL(s, zpi.unique);
```

Comment:
you should also add zpi.ns_id =<> to zread_ipset function.

Reply:
I'm going to use the vrf_id and remove ns_id as the discriminator